src/commands/notify.yml

```diff
@@ -48,6 +48,21 @@ parameters:
         Disable to catch initial setup errors. Re-enable to prevent Slack errors from affecting your pipeline.
       type: boolean
       default: true
+  icon:
+    description: |
+      URL for the icon to be displayed with the post
+    type: string
+    default: ""
+  username:
```

Comment:
This would indicate a static value set in the config. What would "poster" relate to? The commit author? We may have an existing environment variable for the intended user captured here

Reply:
I guess I could have said "the username that should author the post", or something similar. 

I'm open to the reality that I couldn't read the docs correctly and jumped to a PR that is party or wholly unneeded. It's no problem if that's the case.

Reply:
The user that authors the post is the "bot" application that you register upon setup. I do not believe this can be adjusted from the message template.